### DISCUSSION TITLE
Attaching a PVC to queryNodes

### DISCUSSION THREAD

**Discussion by rodrigonascimento** [milvus-io/milvus#28809] [Q&A and General discussion]
Hi, 

Is it possible to attach a PVC to the queryNode mounted as /var/lib/milvus/data?

All the best, 

--Rodrigo

**Comment by wyfeng001** [comment]
If you use Milvus Chart, the current volume is emptydir.  The disk of querynode is generally used for secondary caching, and the performance of cloud disk is generally not enough. It is recommended to use local NVME SSD and that's how Zilliz Cloud used.

However, if you want to use PVC, I think can have a try. Just modify this https://github.com/zilliztech/milvus-helm/blob/master/charts/milvus/templates/querynode-deployment.yaml#L180 from `emptydir` to `persistentVolumeClaim` and use you pvc name.  How to use PVC depends on different clouds, such as EKS you need to install EBS CSI Driver, etc
